packages/eslint-config-terra/.npmignore

```diff
@@ -0,0 +1,9 @@
+*.log
+target
+reports
+tests/**/__snapshots__/
+aggregated-translations
+dev-site-config
+screenshots
+errorScreenshots
+generated-themes
```

Comment:
These prob don't apply to this package

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/d3e6de1d2d39e3db1a4208e58900a71f34df011e